### ISSUE TITLE
Stripe dependency really needs to be updated

### ISSUE THREAD

**Issue by jaspertandy** [solspace/craft3-freeform#842]
**Describe the bug or issue you're experiencing**

This isn't really a bug but there's nowhere to post in discussions where it might get the right attention so I'm posting here.

The dependent version of stripe-php is ^7.27.3 - the latest stable version of stripe-php is 12.2

The inclusion of Freeform is now blocking us from updating quite an important component of our platform - I have one where I'm stuck on stripe-php 7, and another where I'm now stuck on craft commerce 3 because this old version is in use in Freeform.

What's stopping this library being updated? It's frustrating that we're being blocked by a Freeform dependency we don't even use in Freeform. I realise it's massively out of scope for this conversation but I feel like it'd be far easier for everyone if the Stripe stuff was a plugin (yes, a plugin for a plugin!), or a detected optional install or something.

**Comment by alexmglover** [comment]
I've encountered a similar dependency issue while trying to upgrade Commerce Stripe. I noticed that the targeted version for stripe/stripe-php is 7.27.3, which was released back on 03/18/2020.

I understand maintaining plugins can be difficult, but it would be immensely helpful if dependencies could be kept updated.

**Comment by kjmartens** [comment]
We're going to check into it shortly. Sorry for the trouble caused. 🙂

**Comment by jaspertandy** [comment]
Thank you for looking into it.

**Comment by kjmartens** [comment]
This is resolved now in Freeform 4.1.8 🙂

**Comment by alexmglover** [comment]
Thanks for jumping on this so fast!
